src/test/java/org/broadinstitute/hellbender/tools/exome/gcbias/CorrectGCBiasIntegrationTest.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.tools.exome.gcbias;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.exome.*;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+public class CorrectGCBiasIntegrationTest extends CommandLineProgramTest {
+    // Test meta-parameters:
+    private static final int NUM_TARGETS = 10000;
+    private static final int NUM_SAMPLES = 10;
+
+    private static final File TARGET_FILE = createTempFile("targets", ".tsv");
+    private static final File INPUT_COUNTS_FILE = createTempFile("input", ".tsv");
+    private static final File OUTPUT_COUNTS_FILE = createTempFile("output", ".tsv");
+
+    double[] gcContentByTarget;
+    ReadCountCollection inputCounts;
+
+
+    @BeforeClass
```

Comment:
I really, really like generating test files in the code because it's self-documenting, but I see your point (or at least what I *think* is your point) that reuse is good.  What would you think of making us both happy by extracting static methods for creating test data into some test utilities class (and having BeforeClass simply invoke the backend)?